base/float.jl

```diff
@@ -750,17 +750,30 @@ end
 end
 
 """
-    floatmin(T)
+    floatmin([T])
 
-The smallest in absolute value non-subnormal value representable by the given
-floating-point DataType `T`.
+Return the smallest positive normal number representable by the floating-point
```

Comment:
Given that positive and negative zeros are possible for floats (IEEE at least), this language still doesn't exclude zero. Perhaps the a combination of the original language and your additions, with the clarification of "non-zero" would be better?

> Return the smallest positive, non-zero, and non-subnormal value representable by the given floating-point type `T` (`T = Float64` by default).

Reply:
I say "positive *normal* number" instead of "positive number", which excludes the possibility of any zeros since zeros are not a normal number.

Reply:
That's a fair point, but that wording is also subtle enough that I think it could be easily missed.

Reply:
Hmm, I'd prefer simple and concise wording in this case. It is apparent that this function does not return a zero from examples. So, adding more words looks rather redundant.